CMakeLists.txt

```diff
@@ -286,6 +286,11 @@ if(HAS_GAMS)
     set(PRIMAL_SOURCES ${PRIMAL_SOURCES} "${GAMS_DIR}/apifiles/C/api/gevmcc.c")
     set(PRIMAL_SOURCES ${PRIMAL_SOURCES} "${GAMS_DIR}/apifiles/C/api/palmcc.c")
 
+    if(EXISTS ${GAMS_DIR}/apifiles/C/api/gcmt.c)
+        set(PRIMAL_SOURCES "${GAMS_DIR}/apifiles/C/api/gcmt.c")
+        set(PRIMAL_HEADERS "${GAMS_DIR}/apifiles/C/api/gcmt.h")
```

Comment:
This should extend `PRIMAL_SOURCES` and not overwrite it, see https://github.com/coin-or/SHOT/issues/102#issuecomment-760293997

Reply:
True, my mistake :-)